pkg/openapi/openapi.go

```diff
@@ -97,11 +92,6 @@ func PatchForResourceUpdate(client discovery.CachedDiscoveryInterface, lastSubmi
 		return nil, "", nil, err
 	}
 
-	resources, err := getResourceSchemasForClient(client)
```

Comment:
Will likely need to add a check that refreshes the cache on failure or following a CRD registration. If a CRD is registered after the schemas are cached, the cache won't include that info.

Reply:
Done.